src/main/java/org/broadinstitute/dsm/db/DdpKit.java

```diff
@@ -60,7 +70,46 @@ public void changeCEOrdered(Connection conn, boolean orderStatus) {
                     + " to " + orderStatus);
         }
         catch (Exception e) {
-            throw new RuntimeException("Could not update ce_ordered status for " + this.getDsmKitRequestId(),e);
+            throw new RuntimeException("Could not update ce_ordered status for " + this.getDsmKitRequestId(), e);
+        }
+    }
+
+    public static boolean hasKitBeenOrderedInCE(Connection conn, String kitLabel) {
+        String query = "select k.ce_order from ddp_kit k where k.kit_label = ?";
```

Comment:
can you move the queries out, so that someone looking to find queries doesn't need to go through the whole class and search for them

Reply:
The CF tutorials on the web said not to do that, I'll find a link to it